guide/src/examples/raytrace.md

```diff
@@ -6,15 +6,15 @@
 [code]: https://github.com/rustwasm/wasm-bindgen/tree/master/examples/raytrace-parallel
 
 **This is an unstable and experimental example** of using threads with
-WebAssembly and Rust, culminating in a parallel raytracer demo. Current
-requirements for viewing this demo are:
+WebAssembly and Rust, culminating in a parallel raytracer demo. The browser requirements are:
 
-* Firefox Nightly - other browsers haven't implemented the proposed WebAssembly
-  features yet.
-* `SharedArrayBuffer` is enabled in `about:config` in Firefox
-
-This demo may also break over time as Firefox updates, but we'll try to keep it
+* Firefox Nightly 
+  - `SharedArrayBuffer` is enabled in `about:config` in Firefox
+  - This demo may also break over time as Firefox updates, but we'll try to keep it
```

Comment:
I think this can be removed now as well

Reply:
Removed `This demo may also break over time as Firefox updates, but we'll try to keep it`

Didn't remove the shared array buffer bit since my Firefox 75 can't run the demo (I don't have any flags enabled)